conf/test_full.config

```diff
@@ -14,57 +14,65 @@ params {
 
     // Input data for full size test
     input     = 'https://github.com/nf-core/test-datasets/raw/taxprofiler/samplesheet_full.csv'
-    databases = 'https://github.com/nf-core/test-datasets/raw/taxprofiler/database_full.csv'
+    databases = 'https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/database_full_v1.1.csv'
```

Comment:
I thought that using `_v1.1` was temporary but lets leave it as it is.

Reply:
Well we could but it will break older releases' test profiles... people running 1.0 would get errors saying that tool doesn't exist...

Unless we make it just throw a warning?

Reply:
Do you suggest to have `databases.csv` for all releases and have a warning? Or have different databases for each release?

Reply:
Thinking about it again, Dachshunds (1.0) will always fail even if in 1.1 (Akita) we don't error when the pipeline gets an unrecognised profiler which wouldn't be great...

I think a new database sheet for each release is better

Reply:
Let's do it like that then.

Reply:
OK! Then I resolve this comment, we keep the same as befroe